ci/plugins/mzcompose/hooks/post-command

```diff
@@ -31,17 +31,22 @@ export_cov() {
       > coverage/"$BUILDKITE_JOB_ID"-"$(basename "$1")".lcov
 }
 
+find_profraw() {
+    # Ignore cargo-test files which are unrelated to us
+    find . -name '*.profraw' -and -not -path '*/llvm-cov-target/*' "$@"
```

Comment:
How did those files end up in a CI job that was not running cargo-test? Were they a left-over from a previous CI job that happened to use the same machine? In this case, maybe we should do a better cleanup at job initialization?

Reply:
Yes, it appears that processes from a previous cargo-test can still be running in the background. How can we collect them all and kill them? I hoped that the exiting cargo-test would kill all its child processes, but this seems not to be the case.

Reply:
killall?

Reply:
Done, run is green, hope it's fine now.